regenerate.js

```diff
@@ -869,7 +869,8 @@
 				var otherLowSurrogateEnd = otherLowSurrogates[1];
 				if (
 					lowSurrogateStart == otherLowSurrogateStart &&
-					lowSurrogateEnd == otherLowSurrogateEnd
+					lowSurrogateEnd == otherLowSurrogateEnd &&
+					otherMapping[1].length === 2
```

Comment:
I am not sure about this code change, it is not obvious for me how the proper code change should look like, just random code code after surface looking at the code